packages/headless/README.md

```diff
@@ -1,25 +1,6 @@
 # Coveo Headless
 
-Contributing:
```

Comment:
Byebye!

A lot of the doc readme is highly deprecated, so I'm just referecing Coveo's website Headless doc